converters/fromZigbee.js

```diff
@@ -4512,7 +4904,17 @@ const converters = {
         type: ['commandOn', 'commandOff'],
         convert: (model, msg, publish, options, meta) => {
             const cmd = msg.type === 'commandOn' ? 'on' : 'off';
-            return {click: 'power', action: cmd};
+            return {action: cmd};
+        },
+    },
+    legacy_CCTSwitch_D0001_on_off: {
+        cluster: 'genOnOff',
```

Comment:
Not needed, use already existing converter `command_on` and `command_off` instead